com.unity.cinemachine/Runtime/Behaviours/CinemachineSmoothPath.cs

```diff
@@ -224,6 +224,8 @@ public override Quaternion EvaluateLocalOrientation(float pos)
                 Vector3 fwd = EvaluateTangent(pos);
                 if (!fwd.AlmostZero())
                     result = Quaternion.LookRotation(fwd) * RollAroundForward(roll);
+                
+                result = Quaternion.Inverse(transform.rotation) * result;
```

Comment:
Yes, this was a regression introduced within the last year or so.
However, the fix is not optimal.  Pushing a better fix.